package.json

```diff
@@ -71,7 +71,7 @@
     "docs:acceptApiChanges": "node --max-old-space-size=6144 scripts/check_published_api_changes.js --accept",
     "osd:bootstrap": "node scripts/build_ts_refs && node scripts/register_git_hook",
     "spec_to_console": "node scripts/spec_to_console",
-    "pkg-version": "./dev-tools/get-version.sh"
+    "pkg-version": "node -e \"console.log(require('./package.json').version)\""
```

Comment:
This is a tiny bit scary but I think it's pretty straightforward and safe for the CI workflows. Can we delete `get-version.sh` too?

Reply:
+1 - we should remove `get-version.sh` if we're no longer using.

Reply:
It is done.